libs/application/templates/id-card/src/lib/IdCardTemplate.ts

```diff
@@ -240,6 +292,10 @@ const IdCardTemplate: ApplicationTemplate<
             },
           ],
           actionCard: {
+            tag: {
+              label: 'YAYYY', // TODO
```

Comment:
Should maybe have a better text here, even though it's something that is TODO

Reply:
removed